### ISSUE TITLE
fxmanifest incorrect

### ISSUE THREAD

**Issue by nzsnapshot** [project-error/fivem-typescript-boilerplate#7]
**Current:** 

```
fxversion 'cerulean'
name 'FiveM TypeScript Boilerplate'
author 'Project Error'
game 'gta5'

server_script 'dist/server/**/*.js'
client_script 'dist/client/**/*.js'

```
**Required:**

```
fx_version 'cerulean'
name 'FiveM TypeScript Boilerplate'
author 'Project Error'
game 'gta5'

server_script 'dist/server/**/*.js'
client_script 'dist/client/**/*.js'
```

Difference

fxversion 'cerulean' -> fx_version 'cerulean'

**Comment by emrik00** [comment]
So, whats the difference here?

**Comment by RockySouthpaw** [comment]
One is fxversion and the other is fx_version

**Comment by TasoOneAsia** [comment]
Fixed with 211eeb8a594427297ae9222950ad3ee90d09fe6b
